JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1145,4 +1186,90 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(SEK)
 }
 
-# </editor-fold> HELPER FUNCTIONS BLOCK
\ No newline at end of file
+.descriptivesQQPlot <- function(dataset, options,  qqvar, levelName=NULL) {
+
+  if (options$descriptivesQQPlot && is.null(options$variables))
+    return(list(result=NULL))
+
+  descriptivesQQPlot <- list()
```

Comment:
`descriptivesQQPlot` can already be a jaspPlot here:
```
  title <- qqvar
  if (!is.null(levelName))
    title <- levelName
      
  descriptivesQQPlot <- createJaspPlot(plot=NULL, width=400, aspectRatio=1, title=title)
  if (!is.null(levelName))
    descriptivesQQPlot$dependOn(optionContainsValue=list(variables=var))
```

Reply:
So either the dependency for the plot is set on the container if it's grouped, or on the plot itself if it is not grouped